ducktape/cluster/remoteaccount.py

```diff
@@ -180,17 +180,21 @@ def create_file(self, path, contents):
     def _ssh_quiet(self, cmd, allow_fail=False):
         """Runs the command on the remote host using SSH and blocks until the remote command finishes.
 
-        If it succeeds, there is no output; if it fails the output is printed and the CalledProcessError is re-raised.
+        If it succeeds, there is no output; if it fails the output is printed
```

Comment:
There appear to be 4 methods which all call this which will now return a status instead of None.

Should the comments on those methods be updated to reflect this behavior?